sdk/js/src/Client.ts

```diff
@@ -707,6 +693,30 @@ export class DevCycleClient<
         if (!oldConfig || isDebugUser || oldConfig.etag !== this.config.etag) {
             this.eventEmitter.emitConfigUpdate(config.variables)
         }
+
+        // Update the streaming connection URL if it has changed (for ex. if the current user has targeting overrides)
+        if (config?.sse?.url) {
+            // construct the streamingConnection if necessary
+            if (!this.streamingConnection) {
+                if (!this.options.disableRealtimeUpdates) {
+                    this.streamingConnection = new StreamingConnection(
+                        config.sse.url,
+                        this.onSSEMessage.bind(this),
+                        this.logger,
+                    )
+                } else {
+                    this.logger.info(
+                        'Disabling Realtime Updates based on Initialization parameter',
+                    )
+                }
+            }
+            if (
```

Comment:
should this be an else if? otherwise it might open a new streaming connection and then immediately reconnect it

Reply:
true good point, fixed